collectors/nvd/collectors.py

```diff
@@ -251,6 +253,13 @@ def collect(self, cve: Union[str, None] = None) -> str:
             # create a new snippet with a flaw (if it does not exist) and link them
             snippet = None
             if self.snippet_creation_enabled:
+                # we do not care about historical data
+                published = timezone.make_aware(
+                    dateparse.parse_datetime(item["published_in_nvd"])
+                )
```

Comment:
Is there a way to do the timezone specification on storing instead of on reading?

Reply:
`Snippet.content` is `JSONField`, so having a `datetime` object here would probably be a problem.

Reply:
asdf